### PR TITLE
Load location to homepage

### PR DESCRIPTION
**Pull request by quangtu2208**
=> Title:Load location to homepage

=> Redmine:
https://edu-redmine.sun-asterisk.vn/issues/11870

=> Attachment File:

### CODE REVIEW

**Comment by quangtu2208** [comment]
ready

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -23,3 +23,17 @@
...
+body {
+  background: white;
+}
+
+.review_title {
+  margin: 0 0 20px 0;
+}
+.reviews {
+  padding: 15px 0;
+  border-bottom: 1px solid #EAEAEA;
```
Comment (huuphuc93) [inline app/assets/stylesheets/application.scss]: dùng biến màu

File: app/views/locations/_location.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    </div>
+    <div class="details">
+      <h4 class="d-flex justify-content-between">
+        <span><%= location.name %></span>
+        <span><%= location.total_rate %></span>
+      </h4>
+      <p>
+        <%= t ".view_on_map" %>  |   <%= location.reviews.count %> <%= t ".reviews" %>
+      </p>
+      <%= link_to t(".view"), location_path(location),class: "price-btn" %>
```
Comment (huuphuc93) [inline app/views/locations/_location.html.erb]: sau dấu , cách 1 space ra em

File: app/views/locations/show.html.erb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <div class="row">
   <div class="col-md-12">
-    <h3><%= t ".reviews" %><span><%= @location.total_rate%></span> (<%= @location.reviews.count %>)</h3>
+    <h3><%= t ".reviews" %><span><%= @location.total_rate%></span>
```
Comment (huuphuc93) [inline app/views/locations/show.html.erb]:
```suggestion
    <h3><%= t ".reviews" %><span><%= @location.total_rate %></span>
```

**Comment by huuphuc93** [comment]
gộp commit

**Comment by quangtu2208** [comment]
ready

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -23,3 +23,17 @@
  *= require_self
  */
 
+body {
+  background: white;
```
Comment (huuphuc93) [inline app/assets/stylesheets/application.scss]: dùng biến màu

File: config/routes.rb [hunk 5]
```diff
@@ -16,7 +16,8 @@
   resources :account_activations, only: %i(edit)
   resources :password_resets, except: %i(show index destroy)
 
-  resources :locations, only: %i(index show new) do
+
+  resources :locations do
     resources :reviews, only: [ :create, :destroy]
```
Comment (huuphuc93) [inline config/routes.rb]: %i() thay cho []

**Comment by quangtu2208** [comment]
ready

**Pull request merged**